bio_phil/webapp/templates/img_carousel_test.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <body>
+        
+        {% for i in image_list %}
+        <img src = "{{i.img.url}}" width = "300px">
```

Comment:
You can make use of the static tag in django to access resources. Just make sure that its included in the static folder. So you wont need to make a url in `urls.py` anymore for the `/media/images/…`

Here's the docs for how to do that: https://docs.djangoproject.com/en/2.1/howto/static-files/